com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TextureUtil.java

```diff
@@ -216,6 +221,24 @@ public static TextureProfile getTextureProfileByPath(TextureProfiles textureProf
 
         return null;
     }
+    private static String textureProfilesExt = TextureProfilesBuilder.class.getAnnotation(BuilderParams.class).outExt();
```

Comment:
not sure if it's a good idea, on the other hand it's not a runtime value, so I think it's ok to cache it like this